src/data/faq.json

```diff
@@ -579,9 +579,9 @@
                         "anchor": "new_device",
                         "active": true,
                         "textblock": [
-                            "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification API of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days after you install the Corona Alert app on the new device. Please note the following instructions:",
+                            "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification system of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days after you install the Corona-Warn-App on the new device. Please note the following instructions:",
                             "<ul><li>Your old device must be able to perform regular checks for possible encounters. Therefore, leave the device permanently turned on and permanently connected to the Internet.</li><li>Keep the old device in a stationary location, e.g., in a specific place in your apartment. If possible, place the device in listening or visual range so that you are alerted early by a notification of the app.</li><li>Actively access the Corona-Warn-App on the old device at least once a day and check your risk status there.</li></ul>",

```

Comment:
Instead of "in your apartment" 
use
"at home"
which is more general.

Suggest instead of
"in listening or visual range"
use
"where you can hear or see it"
The original text is not wrong. The suggestion is a more natural way to express the same thing in English.

Reply:
"at home" might be too general to describe a specific place, so I left that as it is. Same for the german entry.

I used your second recommedation though.